src/components/TextInput/BaseTextInput.js

```diff
@@ -38,8 +38,16 @@ class BaseTextInput extends Component {
     }
 
     componentDidMount() {
+        if (!this.input) {
+            return;
+        }
+
+        if (this.props.exposeValueToRef) {
+            this.props.exposeValueToRef(this.input, this.value);
```

Comment:
I could be mistaken but I think there's still a slight inconsistency with this strategy. I'm not sure how much it matters, but on web `this.input.value` will be set in the `render()`. On mobile it would be set after the component mounts. Should we make sure it happens in the same lifecycle for consistency or does it not matter?

Reply:
That makes sense. I'll resolve this since we are still discussing how to best solve this issue.